inspire_utils/name.py

```diff
@@ -46,12 +46,14 @@ def _prepare_nameparser_constants():
     Configuration is the same for all names (i.e. instances).
     """
     constants = Constants()
-    roman_numeral_suffixes = [u'v', u'vi', u'vii', u'viii', u'ix', u'x',
```

Comment:
did you remove `v` on purpose?

Reply:
Yes, `v` is in suffix_not_acronyms a;ready.
I also added `xi` as it was missing and logic suggests that it should be added if we have `x` and `xii` ;)